include/net/net_pkt.h

```diff
@@ -1589,7 +1589,7 @@ size_t net_pkt_remaining_data(struct net_pkt *pkt);
 /**
  * @brief Update the overall length of a packet
  *
- * @details Unlike net_pkt_pull() below, this does not take packet cursor
+ * @details Unlike net_pkt_remove() below, this does not take packet cursor
```

Comment:
I think your previous suggested name net_pkt_shrink() was better. The remove word sounds like we are removing the net_pkt.

Reply:
ok